api/tests/core/finance/test_api.py

```diff
@@ -1810,7 +1810,7 @@ def test_generate_invoice_file():
         cashflows=[cashflow1],
     )
 
-    # The file should contains only cashflow from the current batch of invoice generation.
+    # The file should contains only cashflow from the selected batch.
```

Comment:
[typo] Tu peux en profiter pour corriger les fautes. :) 
```suggestion
    # The file should contain only the cashflow from the selected batch.
```